app/views/devise/confirmations/new.html.erb

```diff
@@ -0,0 +1,16 @@
+<h2>Resend confirmation instructions</h2>
```

Comment:
We shouldn't need any of the devise views - can just use the defaults as they will be overridden eventually by some sort of template mailer most likely. Did you need to customize something?

Reply:
see:
app/views/devise/mailer/reset_password_instructions.html.erb

Reply:
OK let's remove the other views then and add a TODO comment in the reset password mailer html to remove once we add in templates